2023-06-22 01:09:23,330 TADA INFO starting test `ldmsd_auth_test`
2023-06-22 01:09:23,330 TADA INFO   test-id: 7a1c17a21021d3c50961260a68b97618375984a69d0c567770c0792b844a7ba2
2023-06-22 01:09:23,330 TADA INFO   test-suite: LDMSD
2023-06-22 01:09:23,330 TADA INFO   test-name: ldmsd_auth_test
2023-06-22 01:09:23,330 TADA INFO   test-user: narate
2023-06-22 01:09:23,330 TADA INFO   commit-id: 564784aadef90522c9e94e7797b394d462daa25d
2023-06-22 01:09:23,331 __main__ INFO -- Get or create the cluster --
2023-06-22 01:09:41,567 __main__ INFO -- Start daemons --
2023-06-22 01:10:28,165 __main__ INFO ... wait a bit to make sure ldmsd's are up
2023-06-22 01:10:33,312 TADA INFO assertion 1, root@agg-2(dom3) ldms_ls to agg-2:10000: see all sets, passed
2023-06-22 01:10:33,429 TADA INFO assertion 2, user@agg-2(dom3) ldms_ls to agg-2:10000: see only meminfo, passed
2023-06-22 01:10:33,561 TADA INFO assertion 3, root@headnode(dom4) ldms_ls to agg-2:10001: see all sets, passed
2023-06-22 01:10:33,686 TADA INFO assertion 4, user@headnode(dom4) ldms_ls to agg-2:10001: see only meminfo, passed
2023-06-22 01:10:33,793 TADA INFO assertion 5, root@headnode(dom4) ldms_ls to agg-11:10000: unexpected result: ['Thu Jun 22 01:10:33 2023:    ERROR: auth_munge: munge_decode() failed. Invalid credential', 'Connection failed/rejected.'], failed
2023-06-22 01:10:33,793 TADA INFO test ldmsd_auth_test ended
